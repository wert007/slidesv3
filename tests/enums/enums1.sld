enum YesNo {
    Yes, // = 0,
    No, // = 1,
}

func main() {
    print(YesNo.Yes);
    print(YesNo.No);
}