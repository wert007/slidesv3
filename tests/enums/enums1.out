YesNo.Yes
YesNo.No
= 0
